pty/src/serial.rs

```diff
@@ -149,7 +116,7 @@ impl Child for SerialChild {
         loop {
             std::thread::sleep(Duration::from_secs(5));
 
-            let mut port = self.port.lock().unwrap();
+            let port = self.port.lock().unwrap();
```

Comment:
Since serial2::SerialPort is Send + Sync, I think we can and should remove the Mutex that is used here so that we're not adding an additional bottleneck on top of whatever internal synchronization it is already doing.

Reply:
Yeah you're right